src/components/game/GuessWordListText.tsx

```diff
@@ -0,0 +1,47 @@
+import {ScrollView, StyleSheet, TouchableOpacity} from 'react-native';
+import React from 'react';
+import {CSText} from '../core';
+import {COLORS} from '../../constants/color';
+
+interface GuessWordItemTextProps {
+  wordList: string[];
+  onPress: (index: number) => void;
+}
+
+const GuessWordListText = ({wordList, onPress}: GuessWordItemTextProps) => {
+  return (
+    <ScrollView contentContainerStyle={styles.footerContainer}>
+      {wordList.map((item, index) => (
+        <TouchableOpacity
+          key={index}
+          onPress={() => onPress(index)}
+          style={styles.item}
+          activeOpacity={0.7}>
+          <CSText variant="Bungee" style={styles.text}>
+            {item}
+          </CSText>
+        </TouchableOpacity>
+      ))}
+    </ScrollView>
+  );
+};
+
+export default GuessWordListText;
+
+const styles = StyleSheet.create({
```

Comment:
Create new file GuessWordListText.styles.ts and move styles to file GuessWordListText.styles.ts